src/test/java/it/mulders/mcs/search/SearchCommandHandlerTest.java

```diff
@@ -1,21 +1,15 @@
 package it.mulders.mcs.search;
 
 import it.mulders.mcs.common.Result;
+import it.mulders.mcs.search.printer.DelegatingOutputPrinter;
 import org.assertj.core.api.WithAssertions;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.DisplayNameGeneration;
-import org.junit.jupiter.api.DisplayNameGenerator;
-import org.junit.jupiter.api.Nested;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Could you please not use wildcard imports here?

Reply:
Wildcard imports replaced.